mountd/mountd.c

```diff
@@ -459,7 +447,7 @@ static int manager_process_init(void)
 		setpgid(0, 0);
 
 	if (create_lock_file()) {
-		pr_err("Failed to create lock file: %m\n");
+		pr_err("Failed to create lock file\n");
```

Comment:
The error print will be out in create_lock_file(). e.g. pr_err("Can't create `%s': %s\n", KSMBD_LOCK_FILE, open_m);
and similar print is out here again. Is it okay ?

Reply:
@namjaejeon, I agree that it's redundant in that scenario. However, it does clarify what failed overall if `flock()` or `write()` fails in `create_lock_file()`.